### PR TITLE
andrewId: rrsood; added new file for 36-226; uploaded research report

### PR DESCRIPTION
**Pull request by raunak-sood2003**
# Summary of Changes

- Please indicate the classes you are modifying:
  - 18-100
  - ...

# Checklist:

- [ ] I have added my entries to the `README.md` table of contents in the root directory as links.

Make sure that your contributions:

- [ ] Are accurate and will not mislead the reader
- [ ] Do not encourage or cause academic integrity violations
- [ ] Do not offend professors or unfairly criticize the university

### CODE REVIEW

**Comment by mikinty** [comment]
Can we not include the research paper? I actually really dislike the research paper section in this repo, it doesn't add much value and encourages plagiarism. I'll address it at some point.

**Comment by mikinty** [comment]
Ideally you squash your commits to maybe one or two. You have a lot of commits for not doing match (e.g. adding docx and then deleting it)

File: math_science_reqs/36226.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+-   The homeworks start on the easier side and get harder as the material gets more complex. Make sure that you are on top of the material and get your questions answered as soon as possible.
+    
+-   The recitations are quite hands-off as the TAs allow you to work on the problems on your own for the majority of the time. Be sure to utilize office hours if you don’t find recitation helpful. But whether you go to recitation or not, it is imperative that you do the recitation problems before the exam!
+
+## Resources
+-   Office hours
+    
+-   Recitations
+    
+-   Exam reviews
```
Comment (mikinty) [inline math_science_reqs/36226.md]: Can you elaborate on each one, what they are about, and why they are useful to go to.

Are there any textbooks that are used in the class or extra materials that were helpful? Course website?

**Comment by raunak-sood2003** [comment]
Hello,

I have made the changes that you requested. Please let me know if there is
anything else you would like me to add or change.

Thanks,

Raunak

**Comment by mikinty** [comment]
Yeah one more thing -- can you squash the commits to just 1 or 2 commits. It's currently 9 commits and that includes uploading and deleting the pdf. We don't want that in our version history

**Pull request closed without merging**